src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.Cache.cs

```diff
@@ -100,7 +126,7 @@ private CachedCodeEntry CacheCode(CachedCodeEntryKey key)
             private readonly string _cultureKey;
             private readonly string _pattern;
 
-            public CachedCodeEntryKey(RegexOptions options, string cultureKey, string pattern)
+            internal CachedCodeEntryKey(RegexOptions options, string cultureKey, string pattern)
```

Comment:
This ctor should stay public as the type is internal.

Reply:
Curious, does it matter?

Reply:
It doesn't matter but we should be consistent. I recently spoke with @stephentoub about this and we both (mostly) agree with what Eric Lippert says about this here: https://stackoverflow.com/questions/9302236/why-use-a-public-method-in-an-internal-class

Reply:
Interesting. Thought it should be `internal`. Changed to `public`. BTW, why we have `internal s_cacheSize`? Who would use it in the assembly?

Reply:
correct, s_cacheSize should be private.